docs/installation/resources/example-install.yml

```diff
@@ -0,0 +1,21 @@
+# This file may be committed to your app code.
+version: 1.0
+# This should match the app - package name in your setup.py
+name: hydroexplorer
```

Comment:
By convention, we use the name `my_first_app` for examples in the documentation. Please change for consistency.

Reply:
Resolved